distribution/lib/Standard/Base/0.0.0-dev/docs/api/Errors/Common.md

```diff
@@ -93,7 +93,6 @@
     - to_display_text self -> Standard.Base.Any.Any
     - to_text self -> Standard.Base.Any.Any
 - type Private_Access
-    - Error this_project_name:(Standard.Base.Data.Text.Text|Standard.Base.Nothing.Nothing) target_project_name:(Standard.Base.Data.Text.Text|Standard.Base.Nothing.Nothing) target_method_name:Standard.Base.Data.Text.Text
```

Comment:
- intended incompatible API change
- ironically the `Private_Access` type wasn't _encapsulated enough_
- and its constructor was public!